### DISCUSSION TITLE
Not receiving email to login

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#5670] [Account Issue] [answered]
### Question

Hello team,

I had trouble creating an account on Vercel due to SMS verification not being enabled for my country, Sri Lanka. I filled out a Contact Us form, but haven't heard back. Could you please assist? I also emailed earlier at registration@vercel.com but haven't received any response.

_Here is the case number -> 00188667_

Thanks!

**Comment by swarnava** [comment, marked as answer]
Hi there, our registration team will triage accordingly and get back to you soon.
